src/main/java/com/google/sps/servlets/Utils.java

```diff
@@ -36,7 +38,7 @@
 import javax.servlet.http.HttpServletRequest;
 
 /**
- * Utility class for OAuth flow helpers
+ * Utility class for common functionality
  */
 class Utils {
```

Comment:
should this be public?

Reply:
I think by default this has package visibility so scoped to all the servlets, should we make it public if this is the case

Reply:
Hmm, I think there might be an argument for moving this out of the servlet package, actually, if we want to keep the servlet package to just servlets.

Not super familiar with Java best practice around that.

@sidharthp any thoughts on package structure and protected vs. public?

Reply:
Resolving for now I think this can be changed pretty easily in the future if necessary